### ISSUE TITLE
JSON.parse: unexpected end of data

### ISSUE THREAD

**Issue by scheinercc** [anandthakker/doiuse.com#22]
```javascript
SyntaxError: JSON.parse: unexpected end of data at line 1 column 10051 of the JSON data

fetch/<@http://doiuse.com/bundle.js:104:16
once/<@http://doiuse.com/bundle.js:2371:12
loadFunc@http://doiuse.com/bundle.js:2251:9
readystatechange@http://doiuse.com/bundle.js:2187:13
```

**Comment by taitems** [comment]
Getting the same error on `http://kiandra.com.au/`

![image](https://cloud.githubusercontent.com/assets/234593/6258340/e4a444d2-b81b-11e4-8e6d-21a50b525b80.png)

**Comment by nolanlawson** [comment]
Also seeing this for `http://npm-browser.com`

**Comment by anandthakker** [comment]
Thanks for these reports @nolanlawson and @taitems.  Will look into this when I'm back from traveling.

**Comment by anandthakker** [comment]
@nolanlawson @taitems - Meanwhile, as a temporary workaround, you can paste in your CSS directly, but of course that's much less convenient :-/

**Comment by scunliffe** [comment]
Yeah I got the same error on http://digg.com

**Comment by anandthakker** [comment]
@nolanlawson So what's happening here is that [style-stream][1] is trying to pull down any stylesheets linked by http://npm-browser.com, but when it tries to grab, e.g., http://npm-browser.com/styles/app.css, it just gets redirected back to the homepage instead of getting that styhlesheet.   Same thing happens for me when I point to that stylesheet in my browser.

**Comment by anandthakker** [comment]
@scheinercc Sorry for the delay responding.  The reason it doesn't work when you just paste in the HTML is that, without the original address, doiuse can't figure out where to pull the stylesheets from.  If you put the address instead, it should work.

(Clearly a more informative error message would have been helpful here... :-/ )

**Comment by anandthakker** [comment]
@scunliffe Should be fixed now (via [style-stream@1.1.4])
